### PR TITLE
Add Reinforced Directional Tinted Window

### PR DESCRIPTION
**Pull request by rosieposieeee**
## About the PR

Simply adds a reinforced variant of directional tinted windows.

Content Client_qksxVGBSW0

Content Client_vmqxjf5BgI

## Why / Balance

Sometimes I need a reinforced one!

NOTE: I hereby release all changes made in this PR under MIT license.

## Requirements

- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- add: Added reinforced directional tinted walls.

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml [hunk 1]
```diff
@@ -19,3 +19,25 @@
...
+
+- type: entity
+  id: WindowTintedReinforcedDirectional
+  parent: WindowReinforcedDirectional
+  name: directional reinforced tinted window
+  description: Don't smudge up the glass down there.
+  placement:
+    mode: SnapgridCenter
+    snap:
+    - Window
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml]: inherited from the parent, remove

File: Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml [hunk 2]
```diff
@@ -19,3 +19,25 @@
...
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Windows/directional.rsi #Delta V - Resprite windows
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml:29]: .

File: Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml [hunk 3]
```diff
@@ -19,3 +19,25 @@
...
+    state: tinted_reinforced_window
+  - type: Tag
+    tags:
+      - ForceNoFixRotations
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml]: .

File: Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml [hunk 4]
```diff
@@ -19,3 +19,25 @@
...
+  - type: Icon
+    sprite: DeltaV/Structures/Windows/directional.rsi #Delta V - Resprite windows
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml:32]: .

File: Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml [hunk 5]
```diff
@@ -19,3 +19,25 @@
...
+    state: tinted_reinforced_window
+  - type: Occluder
+    boundingBox: "-0.5,-0.5,0.5,-0.3"
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml]: .

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml (hunk 1 above)
Comment (rosieposieeee) [reply Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml:31]: its parented from the reinforced window, not from the other tinted window

**Comment by FieldCommand** [comment]
I would love to see that the tinted window gets a reinforced variation of this alongside a tinted windoor and a secure tinted windoor version.

File: Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml (hunk 1 above)
Comment (rosieposieeee) [reply Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml]: 👍

File: Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml (hunk 2 above)
Comment (rosieposieeee) [reply Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml:29]: will it work if i remove the 'sprite' part but leave the state?

File: Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml (hunk 3 above)
Comment (rosieposieeee) [reply Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml]: 👍

**Comment by rosieposieeee** [comment]
> I would love to see that the tinted window gets a reinforced variation of this alongside a tinted windoor and a secure tinted windoor version.

maybe in another PR for the fullsize, i don't personally see the need since there's already walls

for the tinted windoors, tragically the way access works requires you to make one billion copies of all the doors. which would not be great for something that might get used in one placeever

**Review by deltanedas — APPROVED** [review]

**Pull request merged**